compiler/ballerina-lang/src/main/java/io/ballerina/projects/JBallerinaBackend.java

```diff
@@ -530,6 +534,63 @@ private Path emitExecutable(Path executableFilePath) {
         return executableFilePath;
     }
 
+    private Path emitGraalExecutable(Path executableFilePath) {
+        // Run Create Executable
+        Manifest manifest = createManifest();
+        Collection<JarLibrary> jarLibraries = jarResolver.getJarFilePathsRequiredForExecution();
+
+        try {
+            assembleExecutableJar(executableFilePath, manifest, jarLibraries);
+        } catch (IOException e) {
+            throw new ProjectException("error while creating the executable jar file for package '" +
+                    this.packageContext.packageName().toString() + "' : " + e.getMessage(), e);
+        }
+
```

Comment:
Can't we call the `emitExecutable` method instead of repeating this code?

Reply:
I called the emitExecutable method